docs/en/latest/tutorials/protect-api.md

```diff
@@ -37,7 +37,7 @@ This represents the configuration of the plugins that are executed during the HT
 
 :::note
 
-If [Route](../terminology/route.md), [Service](../terminology/service.md), [Plugin Config](../terminology/plugin-config.md) or Consumer are all bound to the same for plugins, only one plugin configuration will take effect. The priority of plugin configurations is: Consumer > Route > Plugin Config > Service. At the same time, there are 6 stages involved in the plugin execution process, namely `rewrite`, `access`, `before_proxy`, `header_filter`, `body_filter` and `log`.
+If [Route](../terminology/route.md), [Service](../terminology/service.md), [Plugin Config](../terminology/plugin-config.md) or Consumer are all bound to the same for plugins, only one plugin configuration will take effect. The priority of plugin configurations is: `Consumer` > `Route` > `Plugin Config` > `Service`. Plugins configured in `Global Rules` will be executed first at all times. At the same time, there are 6 stages involved in the plugin execution process, namely `rewrite`, `access`, `before_proxy`, `header_filter`, `body_filter` and `log`.
```

Comment:
```suggestion
If [Route](../terminology/route.md), [Service](../terminology/service.md), [Plugin Config](../terminology/plugin-config.md) or Consumer are all bound to the same for plugins, only one plugin configuration will take effect. The priority of plugin configurations is: `Consumer` > `Consumer Group` > `Route` > `Plugin Config` > `Service`. Plugins configured in `Global Rules` will be executed first at all times. At the same time, there are 6 stages involved in the plugin execution process, namely `rewrite`, `access`, `before_proxy`, `header_filter`, `body_filter` and `log`.
```

Reply:
This entire block of note is explained in:

* https://apisix.apache.org/docs/apisix/next/terminology/plugin/#plugins-merging-precedence
* https://apisix.apache.org/docs/apisix/next/terminology/plugin/#plugins-execution-order
* https://apisix.apache.org/docs/apisix/next/terminology/plugin/#plugins-execution-lifecycle

I suggest we link to content in these links instead of update anomaly.